Rename to 'The Hallow' internally.

vini2003 opened the pull request:
Initial PR, on hold until currently open PRs are closed.

Missing fixes:

- ~~`README.md`~~
- `de_de.json`
- `en_au.json`
- `en_gb.json`
- `en_nz.json`
- `en_us.json`
- `es_es.json`
- `fr_fr.json`
- `pt_br.json`
- `zh_cn.json`

vini2003 committed:
IT'S RENAME TIME BABY

Shnupbups commented on README.md:
```diff
@@ -1,5 +1,5 @@
 # The Hallow
-Fabric Community mod for Hacktoberfest! Formerly known as Spooky Time.
+Fabric Community mod for Hacktoberfest! Formerly known as The Hallow.
```
no

Shrimpstress replied:
nay

vini2003 replied:
In case you didn't notice it:

    Missing fixes:
    - `README.md`

Shrimpstress replied:
To be fair, I assumed this was in relation to before any changes were made at all, not changes made by this PR.

vini2003 replied:
Please review the new `README.md`.

vini2003 committed:
Hippity hoppity README is The Hallow's property!

vini2003 committed:
fabric.mod.json's description is no longer a rebel.

vini2003 committed:
Draylar fixed his thing!

vini2003 committed:
Fix hidden merge conflicts.

vini2003 committed:
Blame Drayl.ar

vini2003 committed:
Altar & Pillar model update.

vini2003 commented:
I'm going to fucking murder someone.

Shnupbups committed:
some things

- Changed Spooky/Hallowed Cactus to Restless Cactus
- Changed Spooky/Hallowed Trumpet to just Trumpet
- Changed actual variable names
- Changed the name of the dimension itself
- FIxed Mummy spawning
- Fixed Hallowed Ore texture being 16x17

vini2003 committed:
Merge pull request #1 from FabricCommunity-Old/master

some things
